prog/download_boot_image.rb

```diff
@@ -34,8 +34,8 @@ def sha256_sum
     hashes = {
       ["ubuntu-jammy", "x64", "20240319"] => "304983616fcba6ee1452e9f38993d7d3b8a90e1eb65fb0054d672ce23294d812",
       ["ubuntu-jammy", "arm64", "20240319"] => "40ea1181447b9395fa03f6f2c405482fe532a348cc46fbb876effcfbbb35336f",
-      ["almalinux-9.3", "x64", "20231113"] => "6bbd060c971fd827a544c7e5e991a7d9e44460a449d2d058a0bb1290dec5a114",
-      ["almalinux-9.3", "arm64", "20231113"] => "a064715bc755346d5a8e1a4c6b1b7abffe4de03f1b0584942d5483ed32aafd67",
+      ["almalinux-9", "x64", "20231113"] => "6bbd060c971fd827a544c7e5e991a7d9e44460a449d2d058a0bb1290dec5a114",
```

Comment:
1. the `url` function needs to be updated (it seems your PR is not based on the up to date `main` branch)
2. What do you think about changing the version from `20231113` to `9.3-20231113`? Otherwise we need to make sure minor versions don't conflict in release dates.

Reply:
For 2, I think date is ok and every new release increases the minor & there is 1-to-1 mapping between minors and dates.

Also, just date is preferred since it is easier to sort & select the latest version.